AirCasting/Dashboard/DashboardView.swift

```diff
@@ -59,9 +59,9 @@ struct DashboardView: View {
         .navigationBarTitle(Strings.DashboardView.dashboardText)
         .navigationBarHidden(true)
         .onChange(of: selectedSection, perform: { newValue in
-            self.selectedSection_.selectedSection = newValue
+            self.selectedSectionNeededForSwipingImplementation.selectedSection = newValue
         })
-        .onChange(of: selectedSection_.selectedSection , perform: { newValue in
+        .onChange(of: selectedSectionNeededForSwipingImplementation.selectedSection , perform: { newValue in
             self.selectedSection = newValue
         })
```

Comment:
This onChange where one property overrides the other, so that they always match can be deleted I think. Please tell me 😄